fontbro/font.py

```diff
@@ -213,11 +213,14 @@ def __init__(self, filepath, **kwargs):
         super().__init__()
 
         self._filepath = None
+        self._fileobject = None
         self._kwargs = None
         self._ttfont = None
 
         if isinstance(filepath, (Path, str)):
             self._init_with_filepath(str(filepath), **kwargs)
+        elif hasattr(filepath, "read"):
```

Comment:
What you think about doing a more straight check like `isinstance(filepath, io.IOBase)`?

Reply:
A file-like object doesn't necessarily inherits from `IOBase`. From example, [Django's `File` object](https://docs.djangoproject.com/en/4.2/ref/files/file/#django.core.files.File) is not an instance of `IOBase`:

```python
from io import IOBase
from django.core.files import File

f = File(b"")
isinstance(f, IOBase)

# False
```